test/PowerShellEditorServices.Test.Shared/References/FunctionReference.ps1

```diff
@@ -0,0 +1,24 @@
+function BasicFunction {}
```

Comment:
I had to make this a file so that the test can get a `ScriptFile`... which has the `References` field which runs the new cached `ReferenceVisitor`.